pkg/model/cluster.go

```diff
@@ -58,10 +59,11 @@ type clusterModel struct {
 	replicasActive      simulator.ThroughStock
 	replicasTerminating ReplicasTerminatingStock
 	replicasTerminated  simulator.SinkStock
-	requestsInBuffer    simulator.ThroughStock
-	requestsFailed      simulator.SinkStock
-	kubernetesClient    kubernetes.Interface
-	endpointsInformer   corev1informers.EndpointsInformer
+	//TODO think how to rename it
+	requestsInBuffer  simulator.ThroughStock
```

Comment:
How about `requeststInRouting`?

Reply:
Renamed it in https://github.com/pivotal/skenario/pull/67/commits/dfd4d60851eea3b87cc9d7e4dc950b95bc92c495 commit